doc/source/release/release_notes.rst

```diff
@@ -8,7 +8,8 @@
 Release Notes
 *************
 
-.. release:: upcoming
+.. release:: 1.0.8
```

Comment:
Can you remove the changes from this file please?

Reply:
Sorry, i thought i ll find timespace to finish this. Thank you for pushing further.